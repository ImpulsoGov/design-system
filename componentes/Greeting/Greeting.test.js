import { render, screen } from "@testing-library/react";
import { Greeting } from "./";

const COMPONENT = "Greeting";
const scenarios = [
  {
    cargo: "Impulser",
    municipio_uf: "BA",
    greeting: "Boas-vindas",
    nome_usuario: "Carmen",
    texto: "",
  }
];

describe(`Componente: ${COMPONENT}`, () => {
  it("deve renderizar corretamente", () => {
    render(<Greeting {...scenarios[0]} />);

    const element = screen.getByTestId(COMPONENT);

    expect(element).toMatchSnapshot();
  });
});
