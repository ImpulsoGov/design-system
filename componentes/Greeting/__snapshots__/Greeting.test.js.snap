// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente: Greeting deve renderizar corretamente 1`] = `
<div
  class="Greeting"
  data-testid="Greeting"
>
  <div
    class="GreetingcargoMunicipio"
  >
    IMPULSER
     Â· 
    BA
  </div>
  <div
    class="GreetingNome"
  >
    Boas-vindas
    , 
    Carmen
  </div>
  <div
    class="GreetingTexto"
  />
</div>
`;
